src/process-effects.js

```diff
@@ -17,6 +17,9 @@ module.exports = tagFunction => {
 
 	// create the component, which will save new effects to the effect queue
 	const tagResult = tagFunction()
+	if (tagResult === undefined) {
+		throw new Error('Tram-One: no element returned from tag, is anything being returned?')
+	}
```

Comment:
Error to throw when a component does not return an element.
Previously, this would throw the following error, because no DOM object was being returned.
```
"Cannot set property 'tram-tag-new-effects' of undefined"
```